### PR TITLE
 testing for input for a model

### PR DESCRIPTION
**Pull request by 090997**
this class is testing for the scheduler model

1. test function: get_input_clashes

- case 1 : request_id = "322"
- test result:
image
2. test function: get_vehicle_time

- case 2: vehicle_id = "369"
- test result:
image

3. test function: get_vehicle_list

- case 3:request_id= "322"
- test result
image

4. test function: get_position_list

- case 4: vehicle_id="369"
- test result
image

5. test function: get_position_list_all

- case 5: request_id="322"
- test result
image

### CODE REVIEW

File: services/optimiser/input_processing.py
Comment (purple-waves) [inline services/optimiser/input_processing.py:1]: This file should not have any tests in it. They should all be in your separate testing file.

File: services/optimiser/testing_for_scheduler.py
Comment (purple-waves) [inline services/optimiser/testing_for_scheduler.py:31]: Please add comments to all of the test cases describing what the expected output is. This will help future maintainers work out what you were actually testing.

File: services/optimiser/testing_for_scheduler.py
Comment (purple-waves) [inline services/optimiser/testing_for_scheduler.py:42]: These functions need to have consistent spacing between them. PyCharm should give you suggestions (yellow squiggly lines) for the appropriate spacing.

**Review by purple-waves — CHANGES_REQUESTED** [review]
From the description, the tests look pretty good. The code needs some improvements in style and where it is located.

File: services/optimiser/testing_for_scheduler.py
Comment (Xinyun-Zhou) [reply services/optimiser/testing_for_scheduler.py:31]: If you are unsure about how to write the comments, you can read my update and I hope it will help.

File: services/optimiser/testing_for_scheduler.py
Comment (Xinyun-Zhou) [inline services/optimiser/testing_for_scheduler.py:1]: The path from my update is FireApp3.0-backend/services/optimiser/testing /test_shift.py, would you mind renaming the file This will help us to work on the same page. Thank you!

**Review by Xinyun-Zhou — CHANGES_REQUESTED** [review]
It is nice that there are no bugs in these implemented features. Please rename the file and add some comments to help our further development, thanks!

**Comment by 090997** [comment]
> 



> Thank you for the comments! I'm not sure if it is visible on the changes files, just would like to double-check that have you renamed the testing file?

yes，I renamed my test file

**Review by Xinyun-Zhou — APPROVED** [review]
Thanks for fixing up!

**Review by purple-waves — APPROVED** [review]
lgtm, thanks for the changes

**Comment by purple-waves** [comment]
Remember to rebase before you try to merge

**Pull request merged**
